src/models/stores/serial.ts

```diff
@@ -43,14 +43,14 @@ export class SerialDevice {
   }
 
   public async requestAndSetPort(){
-    // Filter any local devices so we only see arduino uno and compatible in choices
-    const filters = [
-      { usbVendorId: 0x2341, usbProductId: 0x0043 },
-      { usbVendorId: 0x2341, usbProductId: 0x0001 }
-    ];
+    // Removing filters so that non-Arduino brand boards will show
```

Comment:
Seems worth leaving in the original comment to explain what this does, and maybe worth adding a TODO to bring it back with more options in the future. Could having no limits here cause problems if students have random equipment plugged in?

Reply:
The user headache we want to avoid is when someone by mistake clicks on another device open for a connection (e.g. a printer ?).  I've replaced the comment and added a message to the dialog to help people identify the right device.